### PR TITLE
Parse Log entries

### PR DESCRIPTION
**Pull request by cherijs**
Parse log entries and split into smaller parts, link to pass object
Screenshot 2023-07-09 at 22 13 40

### CODE REVIEW

**Comment by patroqueeet** [comment]
good morning @cherijs very well done! excellent improvement. would adding a filter for `date`, `type`, `status` make sense now?

**Comment by cherijs** [comment]
> good morning @cherijs very well done! excellent improvement. would adding a filter for `date`, `type`, `status` make sense now?

I added list_filter = ("status", "task_type", "pass_type_identifier")  already :) 
but you can still add date_hierarchy and created_at filter

Screenshot 2023-07-10 at 10 01 09

File: django_walletpass/classviews.py [hunk 1]
```diff
@@ -27,6 +27,10 @@ def get_pass(pass_type_id, serial_number):
     return get_object_or_404(Pass, pass_type_identifier=pass_type_id, serial_number=serial_number)
 
 
+def web_service_root_view(request):
```
Comment (patroqueeet) [inline django_walletpass/classviews.py]: this is not a classview. should either inherit from `View` or in views.py I assume. actually, what is the intention of this view? @cherijs

File: django_walletpass/admin.py [hunk 2]
```diff
@@ -1,6 +1,88 @@
...
+class PassAdmin(admin.ModelAdmin):
+    list_display = ("serial_number", "updated_at", "pass_type_identifier", "wallet_pass_")
+    search_fields = ("serial_number", "pass_type_identifier", "authentication_token", "data")
+    list_filter = ("pass_type_identifier", "updated_at")
+    date_hierarchy = "updated_at"
+    readonly_fields = ("wallet_pass_", "updated_at")
+
+    def wallet_pass_(self, obj: Pass):
+        if obj.data:
+            svg_icon = '<svg width="13" height="16" viewBox="0 0 13 16" fill="none" xmlns="http://www.w3.org/2000/svg">' \
```
Comment (patroqueeet) [inline django_walletpass/admin.py:54]: what is this icon for? how would it look like? @cherijs

Comment (cherijs) [reply django_walletpass/admin.py:54]: Screenshot 2023-07-10 at 10 50 56

File: django_walletpass/classviews.py (hunk 1 above)
Comment (cherijs) [reply django_walletpass/classviews.py]: However, it seems that this view should be taken out. 

I understood when this error came about not being able to open web_service_url:
`WALLETPASS["SERVICE_URL"] = 'https://xxx/api/passes/'`

`[2023-07-09 12:18:47 +0300] Register task (for device 6d07103565a61da35b0cb6047736518a, pass type pass.id.lolo.tickets, serial number R1e7lciJx5K_j6_m4iNouABbUXY; with web service url https://xxx/api/passes/) encountered error: Unexpected response code 404`

but i was wrong...

this error was about not being able to open paass `R1e7lciJx5K_j6_m4iNouABbUXY`  not SERVICE_URL root

File: django_walletpass/admin.py (hunk 2 above)
Comment (alexandernst) [reply django_walletpass/admin.py:54]: I'll copy&paste what I wrote in the other PR. I don't know how I feel about hardcoding the content of a svg file here. Can we maybe load this from a file?

**Comment by alexandernst** [comment]
Is this ready to merge?

**Comment by patroqueeet** [comment]
> Is this ready to merge?

not yet, I believe. the SVG might be better in a static file

**Comment by alexandernst** [comment]
@patroqueeet if you're feeling brave enough, maybe clone cherijs's branch and apply the change to the svg?

**Comment by alexandernst** [comment]
This was merged in #29

**Pull request closed without merging**